### PR TITLE
Update Twitter Documentation to Mention API pricing 

### PR DESCRIPTION
**Pull request by CaenJones**

### CODE REVIEW

**Comment by CaenJones** [comment]
Ishan, just a quick recommendation for the pr process:
image

**Comment by CaenJones** [comment]
My PR's would pass all the time!

File: Admin's Manual.md [hunk 1]
```diff
@@ -121,9 +121,14 @@ discord:
     compact_mode: true # optional: disables avatars so messages take up less vertical space
 ```
 
-#### Using the Twitter integration
+#### Using the X integration
+Devzat supports sending updates about who is online on the X social media platform. However, because of the switch from free to paid API keys, This feature no longer is no longer updated.
```
Comment (quackduck) [inline Admin's Manual.md:125]: Why do you think it won't be updated? It might be.

File: Admin's Manual.md [hunk 2]
```diff
@@ -121,9 +121,14 @@ discord:
...
 
-Devzat supports sending updates about who is online to Twitter. You need to make a new Twitter app through a [Twitter developer account](https://developer.twitter.com/en/apply/user)
+![API Key Pricing](https://global.discourse-cdn.com/twitter/original/3X/b/4/b42b6b7e4db2db3f51db70b439e9f111105db04d.png)
```
Comment (quackduck) [inline Admin's Manual.md:127]: We can let people find pricing themselves. Simply mentioning that the api is now paid and then linking to the new app page is enough

**Comment by quackduck** [comment]
let's keep calling twitter twitter because X is cringe

File: README.md [hunk 3]
```diff
@@ -130,7 +130,7 @@ The rest
 
 ## Integrations
 
-When self-hosting an instance, Devzat can integrate with Slack and/or Discord to bridge messages, and Twitter to post new-user announcements. 
+When self-hosting an instance, Devzat can integrate with Slack and/or Discord to bridge messages, and use the X social-meida platform to post new-user announcements (now requires payment).
```
Comment (quackduck) [inline README.md:133]:
```suggestion
When self-hosting an instance, Devzat can integrate with Slack and/or Discord to bridge messages, and Twitter to post new-user announcements.
```
We don't need to say that it requires payment here. It feels awkward to place that in an intro to the integrations.

File: Admin's Manual.md (hunk 1 above)
Comment (CaenJones) [reply Admin's Manual.md]: changed that

File: Admin's Manual.md [hunk 4]
```diff
@@ -122,8 +122,7 @@ discord:
 ```
 
 #### Using the Twitter integration
-
-Devzat supports sending updates about who is online to Twitter. You need to make a new Twitter app through a [Twitter developer account](https://developer.twitter.com/en/apply/user)
+Devzat supports sending updates about who is online on Twitter. Start by creating a new app through a [Twitter developer account](https://developer.twitter.com/en/apply/user) (note: Twitter's API is now paid).
```
Comment (quackduck) [inline Admin's Manual.md:125]:
```suggestion
Devzat supports posting updates about who is online to Twitter. Start by creating a new app through a [Twitter developer account](https://developer.twitter.com/en/apply/user) (note: Twitter's API is now paid).
```

**Pull request merged**